server/controllers/usage-controller.js

```diff
@@ -185,7 +185,7 @@ exports.getSystemStats = async (req, res) => {
     .where("id", user)
     .then((result) => {
       if (result[0].role != "manager") {
-        console.log(result.role);
+        //console.log(result.role);
```

Comment:
Usually it's seen as a bad practice to commit files containing commented lines of code.